test-utils/state-viewer/src/commands.rs

```diff
@@ -50,8 +48,22 @@ pub(crate) fn state(home_dir: &Path, near_config: NearConfig, store: Arc<Store>)
     }
 }
 
+#[derive(ArgEnum, Debug)]
+pub enum DumpStateMode {
+    /// Produces a single `<near_home_dir>/output.json` which contains all of the state, but needs
```

Comment:
I think we should add an extra argument, and let the user specify the output file. Not everyone wants to output the dump result directly to their home directory.

Reply:
Good idea, thanks.